0.10/contrib/etc/generate_container_user

```diff
@@ -0,0 +1,19 @@
+# Set current user in nss_wrapper
+USER_ID=$(id -u)
+GROUP_ID=$(id -g)
+
+if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"1001" ]; then
+
+    NSS_WRAPPER_PASSWD=/opt/app-root/etc/passwd
+    NSS_WRAPPER_GROUP=/etc/group
+
+    cat /etc/passwd | sed -e 's/^default:/builder:/' > $NSS_WRAPPER_PASSWD
```

Comment:
the assemble container will run as uid 1001 so all this logic will be skipped (nss will not be set up) and since the /etc/passwd contains a valid entry for uid 1001, things will be fine (whoami will reported "default" since that's the entry in /etc/passwd)